pages/license.md

```diff
@@ -2,60 +2,24 @@
-    6.  12.6. Project. "Project" means the W3C Community Group or Business Group for which I executed this CLA.
-    7.  12.7. Specification. "Specification" means any specification developed by the Project, as of the date my Contribution is first included in that specification. W3C shall provide the authoritative mechanisms for the identification of Specifications created by the Project.
-    8.  12.8. W3C CLA RF Licensing Requirements. Subject to Section 5, with respect to my Contributions to the Specification, "W3C CLA RF Licensing Requirements" license shall mean a non-assignable, non-sublicensable license to make, have made, use, sell, have sold, offer to sell, import, and distribute and dispose of implementations of my Contributions in the Specification that:
-
-        1.  12.8.1. shall be available to all, worldwide, whether or not they are W3C Members;
-        2.  12.8.2. shall extend to all Essential Claims owned or controlled by me;
-        3.  12.8.3. may be limited to implementations of the Specification, and to what is required by the Specification;
-        4.  12.8.4. may be conditioned on a grant of a reciprocal RF license (as defined in this policy) to all Essential Claims owned or controlled by the licensee. A reciprocal license may be required to be available to all, and a reciprocal license may itself be conditioned on a further reciprocal license from all.
-        5.  12.8.5. may not be conditioned on payment of royalties, fees or other consideration;
-        6.  12.8.6. may be suspended with respect to any licensee when licensor issued by licensee for infringement of claims essential to implement the Specification or any W3C Recommendation;
-        7.  12.8.7. may not impose any further conditions or restrictions on the use of any technology, intellectual property rights, or other restrictions on behavior of the licensee, but may include reasonable, customary terms relating to operation or maintenance of the license relationship such as the following: choice of law and dispute resolution;
-        8.  12.8.8. shall not be considered accepted by an implementer who manifests an intent not to accept the terms of the W3C CLA RF Licensing Requirements license as offered by the licensor.
-        9.  12.8.9. The RF license conforming to the requirements in this policy shall be made available by the licensor as long as the Specification is in effect. The term of such license shall be for the life of the patents in question.
-
-        I am encouraged to provide a contact from which licensing information can be obtained and other relevant licensing information. Any such information will be made publicly available.
-
-    9.  12.9. You or Your. "You," "you," or "your" means any person or entity who exercises copyright or patent rights granted under this CLA, and any person or entity that person or entity controls.
+## W3C SOFTWARE AND DOCUMENT NOTICE AND LICENSE
```

Comment:
Could you remove the upper case from this? Upper case text is problematic with screen readers.